### DISCUSSION TITLE
通过接口获取ipv6如何设置地址优先级

### DISCUSSION THREAD

**Discussion by CzhBlst** [jeessy2/ddns-go#826] [Q&A]
ipv6接口有多个地址，怎么设置优先级顺序。
我想要设置的优先级顺序为：第三个地址>第二个地址>第一个地址

**Comment by jeessy2** [comment]
根据接口地址的先后顺序来的

**Reply by wanglu58** [reply]
ipv4也是根据接口的先后顺序吗？代码是写在哪个文件下的，可以告诉我吗，我想学习一下老师。
